src/api/search.api.js

```diff
@@ -180,23 +182,32 @@ async function search(queryString = '', lang = '') {
             }
             // reading the main values from the attrs
             const { label: title, detail: description, featureId } = location.attrs
-            // reading coordinates (if defined)
-            const coordinate = []
+
+            let coordinate = []
+            let zoom = location.attrs.zoomlevel
             if (location.attrs.x && location.attrs.y) {
                 coordinate.push(location.attrs.x)
                 coordinate.push(location.attrs.y)
             }
+            if (projection.epsg !== LV95.epsg) {
+                // re-projecting result coordinate and zoom to wanted projection
+                coordinate = proj4(LV95.epsg, projection.epsg, coordinate)
+                zoom = translateSwisstopoPyramidZoomToMercatorZoom(zoom)
```

Comment:
This should not be done only for mercator ?

Reply:
Any output projection that is not LV95 should be reprojected, I'm thinking on a way to link the zoom calculation to the CoordinateSystem, so that we can make it flexible too